roles/network_plugin/calico/templates/cni-calico.conflist.j2

```diff
@@ -37,7 +37,7 @@
       "mtu": {{ calico_mtu }},
 {%- endif %}
       "kubernetes": {
-        "kubeconfig": "{% if calico_version is version('v3.3.0', '>=') %}__KUBECONFIG_FILEPATH__{% else %}{{ kube_config_dir }}/node-kubeconfig.yaml{% endif %}"
+        "kubeconfig": "{% if calico_version is version('v3.3.0', '>=') %}{{ calico_kubeconfig_filepath | default('__KUBECONFIG_FILEPATH__') }}{% else %}{{ kube_config_dir }}/node-kubeconfig.yaml{% endif %}"
```

Comment:
What calico has to do with local volume provisioner?

Reply:
ugh, that is an accident, I should probably close this and open another PR

apologies